profiles/SampleMoonMage-setup.yaml

```diff
@@ -0,0 +1,136 @@
+---
+# https://elanthipedia.play.net/mediawiki/index.php/Moon_mage#Circle_Requirements
+# 4 Magics: Augmentation, Attunement, Arcana, Targeted Magic/Debilitation (Does not include Lunar Magic or Astrology)
+# 4 Survivals: Perception, Athletics, Evasion, Skinning (Does not include Thievery)
+# 3 Lores: Appraisal, Tactics, Mechanical Lore (Does not include Scholarship)
+# 2 Additional: Astrology, Scholarship
+
+# It is recommended to get a target or debilitation spell for your second circle. (e.g. Dinazen Olkar or Calm)
+# You will need a runestone in your inventory to train arcana. Grab one from any donation shelf.
+# You will need to add more magic and survival skills depending on your circle.
+
+# Hunting settings
+training_manager_hunting_priority: true
+training_manager_priority_skills:
+- Heavy Thrown
+- Shield Usage
+
+hunting_info:
+- :zone: rats
+  stop_on:
+  - Heavy Thrown
+  - Shield Usage
+  - Evasion
+  - Skinning
+  - Tactics
+
+# Gear settings
+gear:
+- :adjective: rugged
+  :name: leathers
+  :is_leather: true
+  :hinders_lockpicking: true
+  :is_worn: true
+  :swappable: false
+  :tie_to:
+- :adjective: rugged
+  :name: cowl
+  :is_leather: true
+  :hinders_lockpicking: false
+  :is_worn: true
+  :swappable: false
+  :tie_to:
+- :adjective: target
+  :name: shield
+  :is_leather: false
+  :hinders_lockpicking: true
+  :is_worn: true
+  :swappable: false
+  :tie_to:
+- :adjective: parry
+  :name: stick
+  :is_leather: false
+  :hinders_lockpicking: false
+  :is_worn: true
+  :swappable: false
+  :tie_to:
+- :adjective: brass
+  :name: knuckles
+  :is_leather: false
+  :hinders_lockpicking: true
+  :is_worn: true
+  :swappable: false
+  :tie_to:
+- :adjective: elbow
+  :name: spikes
+  :is_leather: false
+  :hinders_lockpicking: false
+  :is_worn: true
+  :swappable: false
+  :tie_to:
+- :adjective: throwing
+  :name: hammer
+  :is_leather: false
+  :hinders_lockpicking: false
+  :is_worn: false
+  :swappable: false
+  :tie_to:
+
+gear_sets:
+  standard:
+  - rugged leathers
+  - rugged cowl
+  - target shield
+  - parry stick
+  - brass knuckles
+  - elbow spikes
+
+# Combat settings
+buff_spells:
+  Shadows:
+    abbrev: shadows
+    recast: 1
+    mana: 1
+
+dance_skill: Large Blunt
+training_abilities:
+  Stealth: 60
+  Tactics: 30
+  PercMana: 240
+  Hunt: 80
+
+weapon_training:
+  Brawling: ''
+  Heavy Thrown: throwing hammer
+  Large Blunt: throwing hammer
+use_stealth_attacks: true
+
+# Non-combat settings
+exp_timers:
+  Astrology:
+    cooldown: 200
+  Attunement:
+    cooldown: 130
+
+crossing_training:
+- Astrology
+- Augmentation
+- Appraisal
+- Athletics
+- Scholarship
+- Mechanical Lore
+- Sorcery
```

Comment:
It's gonna require a change to crossing-training, you can't specify cambrinth charges in training spells.